Update traces.md (#1562)
open-telemetry/opentelemetry.io: content/en/docs/concepts/signals/traces.md

    ],
}
```
 

> A key Trace component in OpenTelemetry is a Span:

Reviewer: Here and elsewhere: don't capitalize "trace", nor "span".

> A key Trace component in OpenTelemetry is a Span:

Reviewer: End the phrase with a period.

 
A [**Span**](/docs/concepts/observability-primer/#spans) represents a unit of
work or operation. Spans are the building blocks of Traces.
 
We'll dive into Spans in [greater
detail](/docs/concepts/observability-primer/#spans), but let's first look at the
list of OpenTelemetry components that play a part in instrumenting our code:
 
- Tracer
- Tracer Provider
- Trace Exporter
- Trace Context
 
### Tracer
 

> A `Tracer` creates spans, where each span has information about what's happening

Reviewer: ```suggestion
A **tracer** creates spans, where each span has information about what's happening
```

for a given operation; for example a request in a service. Tracers are created

> from `TracerProvider`s. In some languages, a global Tracer is already

Reviewer: ```suggestion
by a **tracer provider**. In some languages, a global tracer is already
```

initialized for you.
 
### Tracer Provider
 

> A **Tracer Provider** (sometimes called `TracerProvider`) is a factory to create
> Tracers. In most applications, a Tracer Provider is initialized once, and the
> Trace Provider's lifecycle matches the application's lifecycle.

Reviewer: ```suggestion
A **tracer provider** is a factory used to create tracers. 
In most applications, a tracer provider is initialized once, and the
tracer provider's lifecycle matches the application's lifecycle.
```

Author: How do we distinguish the descriptive words (eg "tracer provider") vs the actual object name?

 
Initializing a Tracer Provider is typically the first step in tracing with
OpenTelemetry. The Tracer Provider's initialization also includes Resource and
Exporter initialization. In some language SDKs, a global Tracer Provider is

> already initialized for you.

Reviewer: Fix capitalized in this paragraph.

 
### Trace Exporters
 

> **Trace Exporters** send traces to a consumer. The consumer can be standard output
> (std.out) for debugging and development-time, the OpenTelemetry Collector, or

Reviewer: ```suggestion
A **trace exporter** sends traces to a consumer. The consumer can be standard output
for debugging and development-time, the OpenTelemetry Collector, or
```

any open-source or vendor backend of your choice.
 
### Trace Context
 

> The **Trace Context** is metadata about trace **Spans**. It provides correlation between
> Spans, across service and process boundaries. For example, let's say that

Reviewer: ```suggestion
A **trace context** is data about trace spans. It provides correlation between
spans, across service and process boundaries. For example, let's say that
```

Reviewer: @cartermp @melbgirl et al.: note the suggested change of "metadata" to just "data". Does that make sense to you?

Reviewer 2: That makes sense to me.

Author: Yep, makes sense.

ServiceA calls ServiceB, and you want to track that call in a trace. In this case,
OpenTelemetry will use Trace Context to capture the trace ID and current
span from ServiceA, so that Spans created in ServiceB can connect and add to
the trace.
 

> This is known as **Context Propagation**.

Reviewer: ```suggestion
This is known as **context propagation**.
```

 
### Context Propagation
 

> **Context Propagation** is the core concept that enables Distributed Tracing. With

Reviewer: ```suggestion
**Context propagation** is the core concept that enables distributed tracing. With
```
